pkgs/os-specific/linux/iputils/default.nix

```diff
@@ -6,7 +6,7 @@
 with lib;
```

Comment:
I would prefer if the with would be removed and lib put where needed.

Reply:
Sounds reasonable, I've made the necessary changes (but I kept it for meta, as usual).